client/src/app/app.component.ts

```diff
@@ -173,5 +179,14 @@ export class AppComponent implements OnInit {
 
   toggleMenu () {
     this.isMenuDisplayed = !this.isMenuDisplayed
+    this.isMenuWanted = true
```

Comment:
`isMenuChangedByUser` maybe?

Reply:
nice! I am not good at naming variables :sweat_smile: